src/zeit/content/cp/automatic.py

```diff
@@ -224,19 +224,19 @@ def __call__(self):
     def _build_query(self):
         if self.context.is_complete_query:
             query = self.query
+            if self.hide_dupes_clause:
+                query = {'query': {'bool': {
+                    'must': query,
+                    'must_not': self.hide_dupes_clause}}}
```

Comment:
Haha, dann ist `is_complete_query` aber auch schon wieder geschwindelt... 😄

Reply:
Nen besseren Namensvorschlag nehm ich gern... :)

Ich bin auch immernoch nicht so richtig happy mit dieser Umschreiberei, weil es ein ziemliches Gebastel ist (wird im kommenden UI-PR noch mehr), aber das Feature an sich halte ich für sehr sinnvoll, sodass ich es glaub in Kauf nehme.